<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AnnotationGroupUI</class>
 <widget class="QWidget" name="AnnotationGroupUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>649</width>
    <height>595</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>GroupBox</string>
  </property>
  <property name="title" stdset="0">
   <string>Group annotations</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="sizeConstraint">
    <enum>QLayout::SetDefaultConstraint</enum>
   </property>
   <property name="leftMargin">
    <number>9</number>
   </property>
   <item row="12" column="0" colspan="2">
    <widget class="QLabel" name="lblDownArrow">
     <property name="text">
      <string>â†§</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="16" column="0" colspan="2">
    <layout class="QHBoxLayout" name="horizontalLayout" stretch="2">
     <item>
      <widget class="QComboBox" name="comboGroup">
       <property name="statusTip">
        <string/>
       </property>
       <property name="currentText">
        <string/>
       </property>
       <property name="placeholderText">
        <string>select group</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="5" column="0" colspan="2">
    <widget class="QTreeView" name="treeView"/>
   </item>
   <item row="8" column="0">
    <widget class="QPushButton" name="btnPolygonSelection">
     <property name="text">
      <string>polygon selection</string>
     </property>
    </widget>
   </item>
   <item row="6" column="1">
    <widget class="QPushButton" name="btnEditGroup">
     <property name="text">
      <string>edit Group</string>
     </property>
    </widget>
   </item>
   <item row="17" column="0">
    <widget class="QPushButton" name="btnAddTrajectories">
     <property name="minimumSize">
      <size>
       <width>310</width>
       <height>26</height>
      </size>
     </property>
     <property name="text">
      <string>add</string>
     </property>
    </widget>
   </item>
   <item row="11" column="0" colspan="2">
    <widget class="QLineEdit" name="txtTrajectoriesToAdd"/>
   </item>
   <item row="9" column="0">
    <widget class="QLabel" name="lblBulkAdd">
     <property name="text">
      <string>add trajectories</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QPushButton" name="btnExport">
     <property name="text">
      <string>export</string>
     </property>
    </widget>
   </item>
   <item row="6" column="0">
    <widget class="QPushButton" name="btnAddGroup">
     <property name="text">
      <string>add Group</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QPushButton" name="btnImport">
     <property name="text">
      <string>import</string>
     </property>
    </widget>
   </item>
   <item row="7" column="0" colspan="2">
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
   </item>
   <item row="8" column="1">
    <widget class="QPushButton" name="btnReset">
     <property name="text">
      <string>reset</string>
     </property>
    </widget>
   </item>
   <item row="17" column="1">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="lblColorPreview">
       <property name="minimumSize">
        <size>
         <width>34</width>
         <height>0</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="0" colspan="2">
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <property name="sizeConstraint">
      <enum>QLayout::SetDefaultConstraint</enum>
     </property>
     <item>
      <widget class="QCheckBox" name="ckbxShow">
       <property name="toolTip">
        <string>display colored circles on trajectories to show current group assignment</string>
       </property>
       <property name="text">
        <string>show visualization </string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>1</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="toolTip">
        <string>Define the radius (in pixel) of the circles when visualizing groups</string>
       </property>
       <property name="text">
        <string>radius (px):</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="spinRadius">
       <property name="value">
        <number>50</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>btnExport</tabstop>
  <tabstop>btnImport</tabstop>
  <tabstop>treeView</tabstop>
  <tabstop>btnAddGroup</tabstop>
  <tabstop>btnEditGroup</tabstop>
  <tabstop>btnPolygonSelection</tabstop>
  <tabstop>btnReset</tabstop>
  <tabstop>txtTrajectoriesToAdd</tabstop>
  <tabstop>comboGroup</tabstop>
  <tabstop>btnAddTrajectories</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
